player.js

```diff
@@ -0,0 +1,37 @@
+module.exports.appendPlayerRegex = function(command, optional) {
+    var playerRegex = command + "(?: @?([^\\s]+))";
+
+    if (optional) {
+        return new RegExp(playerRegex + "?");
+    }
+
+    return new RegExp(playerRegex);
+};
+
+module.exports.getSlackUser = function(users, message) {
```

Comment:
doesnt need to happen now, but probably lichess user info could be added here... 

OR maybe we move all the lichess stuff into its own js file. in any case, more refactoring and better code organization.